src/components/SignIn.js

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+
+const SignIn = () => {
+  return (
+    <div class="w-full max-w-xs">
```

Comment:
Do all these `class=...` need to be changed to `className=...` or does this now work in React?

Reply:
that is tailwind's form of styling components